install

```diff
@@ -432,6 +432,7 @@ if !mac?
     PATH="#{HOMEBREW_PREFIX}/bin:$PATH"
 - We recommend that you install GCC by running:
     brew install gcc
+- After modifying your shell profile, you may need to restart your session (logout and then log back in) if the brew command isn't found.
```

Comment:
I'd suggest breaking this line somewhere as it is too long.

Reply:
Let's add a line break after `session`.
```
- After modifying your shell profile, you may need to restart your session
  (logout and then log back in) if the brew command isn't found.
```